src/lib/SDK/sdk.ts

```diff
@@ -196,7 +198,9 @@ export class SDK {
       });
     }
 
-    const apiUrl = `${this.apiPath}/accounts/${opts.publicAddress}/assets/nfts`;
+    let apiUrl = `${this.apiPath}/accounts/${opts.publicAddress}/assets/nfts`;
+
+    if (opts.cursor) apiUrl = apiUrl.concat(`?cursor=${opts.cursor}`);
```

Comment:
i don't like the concat , for me it's better to change the `HttpClient` , and allow the get method of this class to have an optional query param as parameter, like what we are doing for the post method of `HttpClient`